packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -2,8 +2,10 @@
   package="com.mobile">
 
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>
```

Comment:
Don't we need to declare camera access as well?

Reply:
the library does not require declare camera. do we need it anyway?

Reply:
I'd do that

Reply:
updated